_posts/0000-01-02-Cyb3rm4n20.md

```diff
@@ -0,0 +1,6 @@
+< ---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
It's very easy to make some words **bold** and other words *italic* with Markdown. You can even [link to Google!](http://google.com)
![hp logo](https://user-images.githubusercontent.com/74148535/99081443-b96ceb80-25ba-11eb-8646-80e81cc49ba2.jpg)

Reply:
yeah.